### DISCUSSION TITLE
Move IRC channel to freenode?

### DISCUSSION THREAD

**Discussion by bigpresh** [GriefPrevention/GriefPrevention#161] [Q&A] [answered]
Hi,

I spotted a link to an IRC channel via Spigot's web-IRC gateway: http://irc.spi.gt/iris/?nick=&channels=GriefPrevention from the project page on spigot.org.

Doesn't look like it gets used much, and SpigotMC's IRC network appears to be tiny.

I wonder if it would be worth using a channel on freenode instead - a far, far larger network, and focussed on open source development.   Might find more people use it, and I'd be happy to idle in it as I'm, unsurprisingly, always on freenode anyway :)

@RoboMWM what do you think?

**Comment by RoboMWM** [comment, marked as answer]
I, on the contrary, hardly use freenode lol. If anything, I've found more bukkit projects to be on espernet (which is where the official bukkit, and formerly the official spigot channels were located).

If anyone wants to bridge chat from other networks with the spigot one, I'm fine with that.

**Comment by bigpresh** [comment]
Fair enough - if you'd like it to stay where it is, that's cool.  I can always add another network in irssi :)
